src/control/cmd/dmg/pretty/pool.go

```diff
@@ -226,12 +263,21 @@ func poolListCreateRowVerbose(pool *control.Pool) txtfmt.TableRow {
 		svcReps = formatRanks(rl)
 	}
 
+	var upgrade string
+
+	if pool.PoolLayoutVer != pool.UpgradeLayoutVer {
+		upgrade = fmt.Sprintf("%d->%d", pool.PoolLayoutVer, pool.UpgradeLayoutVer)
+	} else {
+		upgrade = "None"
+	}
```

Comment:
(style) In Go, it is considered more idiomatic to initialize a variable to its default, and to minimize if/else branches.
```suggestion
	upgrade := "None"
	if pool.PoolLayoutVer != pool.UpgradeLayoutVer {
		upgrade = fmt.Sprintf("%d->%d", pool.PoolLayoutVer, pool.UpgradeLayoutVer)
	}
```